docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
 # Windows build
 
+How to make Neuron compilable in Windows.  
+ 
 ## Pre-requisites
 
-You need to install the following:
+You need to follow the steps:
 
-*  Git: [https://git-scm.com/](https://git-scm.com/)
-*  Chocolatey: [https://chocolatey.org/](https://chocolatey.org/)
-*  Visual Studio with C++: [https://visualstudio.microsoft.com/vs/features/cplusplus/](https://visualstudio.microsoft.com/vs/features/cplusplus/)
-*  `pwsh` (grab the latest PowerShell release on GitHub: [https://github.com/PowerShell/PowerShell/releases/latest](https://github.com/PowerShell/PowerShell/releases/latest))
+### Step 1: (If installing neuron on a Windows Virtual Machine) 
+
+Create a new VM 
+  
+------------------------------------------------------------ 
+
+### Step 2: Install Git 
+
+Git: [https://git-scm.com/](https://git-scm.com/)
+
+Use this: [https://git-scm.com/download/win](https://git-scm.com/download/win)
+
+Note: Do not use: `winget install --id Git.Git -e --source winget` 
+ 
+  
+------------------------------------------------------------ 
+
+### Step 3: Install Chocolatey [https://chocolatey.org/](https://chocolatey.org/) by command line: 
+
+```bash
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1')) 
+```
+
+Ensuring Chocolatey commands are on the path 
+
+Ensuring chocolatey.nupkg is in the lib folder 
+ 
+ 
+------------------------------------------------------------ 
+
+### Step 4: Install Visual Studio with C++ 
+
+Use this: [https://visualstudio.microsoft.com/vs/features/cplusplus/](https://visualstudio.microsoft.com/vs/features/cplusplus/) 
+
+ 
+------------------------------------------------------------ 
+
+### Step 5: Get PowerShell 7 or newer version
+
+We will use `pwsh` command - correspond to PowerShell 7. The PowerShell 5 use `powershell` command.  
+ 
+#### Step 5.A: See PowerShell Version Installed: 
+ 
+`$PSVersionTable` or `get-host|Select-Object`  
```

Comment:
It might be worth clarifying that this must be run inside whatever PowerShell you want to test... so it's not which version is installed per se (because you for example have 5 and 7) but which one you are currently running.